File: database/src/main/kotlin/com/tsobu/ona/database/entities/dataval/PpTzWaP4Entity.java
package com.tsobu.ona.database.entities.dataval;

import lombok.Data;
import lombok.EqualsAndHashCode;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

@Entity
@Table(name = "dataval_pp_tz_wa_p4")
@Data
@EqualsAndHashCode(callSuper = true)
public class PpTzWaP4Entity extends com.tsobu.ona.database.entities.BaseEntity implements {

    @Column(name = "index_p4")
    private Integer indexP4;

    @Column(name = "weed_pic_p4")
    private String weedPicP4;

    @Column(name = "weed_cover_p4")
    private String weedCoverP4;

    @Column(name = "cassava_height_p4")
    private String cassavaHeightP4;

    @Column(name = "weeds_height_p4")
    private String weedsHeightP4;

    @Column(name = "parent_key")
    private String parentKey;

    @Column(name = "control_key")
    private String controlKey;

    @Column(name = "set_of_weed_assessment_p4")
    private String setOfWeedAssessmentP4;

}

File: database/src/main/kotlin/com/tsobu/ona/database/entities/dataval/PpTzWaP4Entity.kt
package com.tsobu.ona.database.entities.dataval

import com.tsobu.ona.database.entities.BaseEntity
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Table

@Entity
@Table(name = "dataval_pp_tz_wa_p4")
class PpTzWaP4Entity : BaseEntity() {
    @Column(name = "index_p4")
    var indexP4: String? = null

    @Column(name = "weed_pic_p4")
    var weedPicP4: String? = null

    @Column(name = "weed_cover_p4")
    var weedCoverP4: String? = null

    @Column(name = "cassava_height_p4")
    var cassavaHeightP4: String? = null

    @Column(name = "weeds_height_p4")
    var weedsHeightP4: String? = null

    @Column(name = "parent_key")
    var parentKey: String? = null

    @Column(name = "control_key")
    var controlKey: String? = null

    @Column(name = "set_of_weed_assessment_p4")
    var setOfWeedAssessmentP4: String? = null
}